srcpkgs/calligra/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'calligra'
 pkgname=calligra
 version=3.2.1
-revision=2
+revision=3
```

Comment:
Why revbump? No packahge content would change?

Reply:
`libkokross.so.17` isn't available in `_2`,